.concourse/pipeline.yaml.gomplate

```diff
@@ -73,11 +73,11 @@ resources:
   type: git
   source:
     branch: {{ $branch }}
-    uri: https://github.com/{{ if has . "kubecf_repository" }}{{ .kubecf_repository }}{{ else }}{{ "cloudfoundry-incubator/kubecf" }}{{ end }}
+    uri: https://github.com/{{ if has $ "kubecf_repository" }}{{ $.kubecf_repository }}{{ else }}{{ "cloudfoundry-incubator/kubecf" }}{{ end }}
 - name: status-{{ $branch }}.src
   type: github-status
   source:
-    repo: {{ if has . "kubecf_repository" }}{{ .kubecf_repository }}{{ else }}{{ "cloudfoundry-incubator/kubecf" }}{{ end }}
+    repo: {{ if has $ "kubecf_repository" }}{{ $.kubecf_repository }}{{ else }}{{ "cloudfoundry-incubator/kubecf" }}{{ end }}
```

Comment:
```suggestion
    repo: {{ default "cloudfoundry-incubator/kubecf" $.kubecf_repository }}
```

Reply:
:warning: gomplate __does not have__ a `default` function. Using this form fails when the value is not se.

Reply:
> warning gomplate **does not have** a `default` function. Using this form fails when the value is not set.